server/alpine.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM frolvlad/alpine-glibc:latest
+
+LABEL varion="1.0" \
+    maintainer="h1dden-da3m0n" \
+    description="A simple TeaSpeak server running on alpine-glibc (amd64)"
+
+ARG TEASPEAK_VERSION="1.4.12"
+
+RUN	apk update --no-cache && apk upgrade --no-cache \
+    && apk add --no-cache \
+         bash ca-certificates wget curl python libnice ffmpeg tzdata \
+    \
+    && mkdir ts && cd /ts \
+    && wget -nv -O TeaSpeak.tar.gz https://repo.teaspeak.de/server/linux/amd64/TeaSpeak-$TEASPEAK_VERSION.tar.gz \
+    && tar -xzf TeaSpeak.tar.gz \
+    && rm TeaSpeak.tar.gz \
+    && curl -sLS https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl \
+    && chmod a+rx /usr/local/bin/youtube-dl
+
+WORKDIR /ts
+
+EXPOSE 9987/udp 10101/tcp 30303
+
+VOLUME ["/ts/logs", "/ts/certs", "/ts/files", "/ts/database"]
```

Comment:
Is it possible to expose single files (`config.yml` would be from importance)

Reply:
yes, that should be possible, yet I haven't seen it done really.
Usually, you have a config folder volume definition, but ill test and see if a single file is fine too.
Alternatively, you can always bind-/volume-mount files/folders even if they are not defined here.
All this Volume declaration does is that it guarantees that the defined locations get mounted, even if the user doesn't specify a mount at runtime. Such a situation would result in anonymously created volumes that get auto mounted to those predefined volume paths. So all that this line does is guaranteeing that the files and folders in the paths listed there won't get written onto the container write layer.